nixos/modules/services/networking/searx.nix

```diff
@@ -65,6 +67,12 @@ in
         '';
       };
 
+      redisCreateLocally = mkOption {
+        type = types.bool;
+        default = false;
+        description = lib.mdDoc "Configure local Redis server for searx.";
```

Comment:
Is this needed for the limiter only or is also used for other purposes? In any case try to explain when it's needed. Also explain that it's a searx-ng only feature.